### PR TITLE
Release devel 2.0 beta16

### PR DESCRIPTION
**Pull request by nskaggs**
Also add note about getting beta from snap store

### CODE REVIEW

File: src/en/temp-release-notes.md [hunk 1]
```diff
@@ -69,72 +71,70 @@ The "state-server" from Juju 1.x becomes a "controller" in 2.0.
...
-| juju space create                     | juju add-space *** |
-| juju space list                       | juju list-spaces *** |
-| juju subnet add                       | juju add-subnet *** |
-| juju ensure-availability              | juju enable-ha *** |
-
-\* the behaviour of destroy-environment/destroy-model has changed, see
-https://jujucharms.com/docs/devel/controllers
-\*\* these commands existed but are now the recommended approach
-\*\*\* this is an alias, but will be the primary command going forward
+    1.25 command                          2.0-beta command
```
Comment (matthewhelmke) [inline src/en/temp-release-notes.md]: Curious why we are reformatting this from an easy-to-read table to a code block that is harder to read...

File: src/en/temp-release-notes.md [hunk 2]
```diff
@@ -819,8 +819,8 @@ relevant sub-commands:
 
 For more information on the new support for channels in the Charm Store
 and how they work, please see our
-[documentation](https://jujucharms.com/docs/devel/authors-charm-store#entities-explained)
-on the subject.
+[documentation](https://jujucharms.com/docs/devel/authors-charm-store
+#entities-explained) on the subject.
```
Comment (matthewhelmke) [inline src/en/temp-release-notes.md]: The entire link needs to be on one line or it breaks the link. This line should not have been changed.

**Comment by matthewhelmke** [comment]
I left a couple of comments. Other than those, this looks good.

File: src/en/temp-release-notes.md (hunk 1 above)
Comment (nskaggs) [inline src/en/temp-release-notes.md]: I'm not sure; I can simply revert the change and we can ask about it in the doc itself.

**Comment by nskaggs** [comment]
Got rid of unwanted stuff, ty.

**Comment by matthewhelmke** [comment]
ty
:beer:

**Pull request merged**